vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -1012,7 +1012,7 @@ namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem
 
                 let stripEndingSemicolon (sb : StringBuilder) =
```

Comment:
Since the fixes issue #3318 we don't really need know whether trimming whitespace is even better fix :-)